mongo_db/item_store.py

```diff
@@ -0,0 +1,203 @@
+import os
+import pymongo
+import datetime
+from status import Status
+from bson.objectid import ObjectId
+from item import Item
+from mongo_db.config import Config
+
+class ItemStore:
+
+    def __init__(self):
+        self.MONGODB_CLIENT = None
+        self.MONGODB_DATABASE = None
+
+    def get_mongodb(self, db_name=Config().MONGO_DEFAULT_DATABASE):
+        """
+        Returns the MongoDB
+
+        Returns:
+            db: The Mongo DB database object.
+        """
+        if self.MONGODB_DATABASE is None:
+
+            if self.MONGODB_CLIENT is None:
+                self.MONGODB_CLIENT = pymongo.MongoClient(
+                    f'mongodb+srv://{Config().MONGO_USER_NAME}:{Config().MONGO_PASSWORD}@{Config().MONGO_HOST}/{db_name}?retryWrites=true&w=majority')
```

Comment:
You could have made the whole connection string a single config variable instead of splitting it up by username/password/host. So that e.g. devs could do `mongodb://localhost/etc`